nlp/2021_06_18_data_augmentation/totw_nlp_dat_aug.ipynb

```diff
@@ -0,0 +1,6046 @@
+            "text/plain": [
+              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=239.0, style=ProgressStyle(description_…"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "application/vnd.jupyter.widget-view+json": {
+              "model_id": "8e447e52a88045c689ea9c6b7edcc622",
+              "version_minor": 0,
+              "version_major": 2
+            },
+            "text/plain": [
+              "HBox(children=(FloatProgress(value=0.0, description='Downloading', max=1109.0, style=ProgressStyle(description…"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "EDSDPa3-o9mT",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 250,
+          "referenced_widgets": [
+            "7a4b13bbfa5f4d91b87988474706db00",
+            "98d6c27b893c4ad2ba9066aeb500bb68",
+            "7732792a9377432e93380df5f58c066b",
+            "bcbed4f85eb14fc19f87d76069b533c0",
+            "fad945675c3440c0aad1240c7c4d734c",
+            "fe7dbbfd8e3547269f3c4e0b1f522571",
+            "62f5957783b14115af9d08868f11bdd1",
+            "695869f024354a14bd8977943e6d12ab",
+            "e1b69dc954bb42a09f7a14d0e49f337a",
+            "482ec3b545e94964b09ccab024fa7761",
+            "5bf6830eb39d40dfb257e82b33e5ddc7",
+            "9d3254bc658c46e591e34fd61ee65c89",
+            "d2766cb35a4f4648be54190b121e3b46",
+            "22c35c0b7c3d4ca39ff0d14d2ca4b841",
+            "8e5b4f0b26bf4f9fa6ef6b7b82ab36ab",
+            "b33932dc798a489eb02c251b1f031760",
+            "1751276e738a4f67be6e246f943cc7f3",
+            "f8d798802cbf455b9f06b5414f5bf925",
+            "f3d85e25d6c9499ba5d8e495e89d72ae",
+            "d5549915e1254a41b7b9dcabcb4aa338",
+            "3ab662c04bb6481fad5f72375461c1b2",
+            "992f5cd153104d0a9c521606eb097794",
+            "6541923d3e2d4c08813c2a218313a4bb",
+            "6332282b65cb4a23bca90b9170d6e40c",
+            "8cea4f40d9b04af199163d4a6378de4d",
+            "7d96e3dcc3274114afa8e5a59f1b9570",
+            "cd8c88b6353e485b9cd33a17c111dc21",
+            "d4fe7f2bf01b4712b187da3aa2b7de28",
+            "dee09c8a70ed44c6aa3642ca695bcfd2",
+            "6c9043c410c44bc592b6d9c6853ff5d8",
+            "c82b139808e94d8da04c2626dfc4c725",
+            "c715a72cba5f4088b8c72932b39a181d"
+          ]
+        },
+        "outputId": "02bba973-ea85-44c0-f5fb-d5d2ab06cca0"
+      },
+      "source": [
```

Comment:
Some extra whitespaces and perhaps a small comment would make this cell more readable IMO